### ISSUE TITLE
Modify maximum values for spring easing parameters

### ISSUE THREAD

**Issue by FrancescoMagalini1** [juliangarnier/anime#824]
Is it possible to modify the maximum values for the parameters of spring easing?
All the parameters have a maximum of 100, but mass and velocity don't need to be that high while stiffness and damping could be higher. I think it would make easier working with other libraries ( [react-spring](https://react-spring-visualizer.com) ) or design tools ( [figma](https://www.figma.com/blog/how-we-built-spring-animations/) ) and also it would be possible to have more alternatives to choose from.

Thank you best regards

**Comment by juliangarnier** [comment]
Good point, will have a closer look at how springs are defined in other libs / platforms.
